src/config/texts.js

```diff
@@ -448,7 +448,13 @@ export const texts = {
     date: 'Datum',
     hideFilter: 'Filter ausblenden',
     resetFilter: 'Filter zurücksetzen',
-    showFilter: 'Filter anzeigen'
+    showFilter: 'Filter anzeigen',
+    sorting: {
+      changeDate: 'Änderungsdatum',
+      creationDate: 'Erstelldatum',
```

Comment:
please see the API, these are called differently. i will update them to `created_datetime` and `updated_datetime`